docs/editors/overview.md

```diff
@@ -38,181 +39,207 @@ functionality.
     <td align="center">Single click</td>
     <td align="center">Single click</td>
     <td align="center">Single click</td>
+    <td align="center">Manual</td>
   </tr>
   <tr>
     <td>Build import</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Diagnostics</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Goto definition</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Goto type definition</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Completions</td>
     <td align="center">✅</td>
     <td align="center">✅*</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Hover</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Hover for selection</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">❌</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Parameter hints</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Find references</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Run/Debug</td>
     <td align="center">✅</td>
     <td align="center"></td>
     <td align="center"></td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
```

Comment:
This one I wonder about. :thinking:  I still have to successfully configure the DAP integration with Metals, but AFAIK it should be possible to make it work, at least the basics (Helix support for it is currently limited anyway). Although it's true that right now it's not prepared out of the box. Maybe @ckipp01 could help here, since he has managed to connect it with Neovim. :slightly_smiling_face:

Reply:
No idea if this will be helpful or not, but the docs for this for nvim-metals is [here](https://github.com/scalameta/nvim-metals/blob/d90ea43ded7ec651606b0a533ae9740083436e58/doc/metals.txt#L1231-L1318). There is for sure a couple steps that was necessary to make this work. We also have some Metals docs [here](https://scalameta.org/metals/docs/integrations/debug-adapter-protocol) that might be helpful, and if you really want a deep dive I wrote an article about this a while back explaining the entire flow [here](https://www.chris-kipp.io/blog/the-debug-adapter-protocol-and-scala).